### ISSUE TITLE
Russian Translation Not Showing on Win 10

### ISSUE THREAD

**Issue by besdar** [NickvisionApps/Parabolic#900]
Hello @nlogozzo! I've just installed your latest version of the application on Windows 10 with Russian locale, but I see your application in English after the installation. Is it possible to change the UI language somehow? 

I've checked the documentation and older issues, but I don't see any information about this. Thank you in advance for your support!

**Comment by nlogozzo** [comment]
> Is it possible to change the UI language somehow?

This is not possible. Parabolic should be seeing your system language and providing that translation.

Can you go to the `Help` menu --> Select `About` --> Click the `2024.10.1` button ? It will copy some information that you can come back here and paste for me

**Comment by besdar** [comment]
@nlogozzo I've copied, but it gave me gibberish

![Screenshot from 2024-10-15 19-57-59](https://github.com/user-attachments/assets/e86a8747-73bd-4e33-b0f9-40b7c7a1e9ed)

here is the text:
```
org.nickvision.tubeconverter
Windows
2024.10.1

Deployment Mode: Local
Locale: ru-RU
Running From: "C:\\Program Files (x86)\\Nickvision Parabolic\\Release"

yt-dlp version "C:\Program" �� ����� ����७��� ��� ���譥�
��������, �ᯮ��塞�� �ணࠬ��� ��� �������� 䠩���.
"C:\Program" �� ����� ����७��� ��� ���譥�
��������, �ᯮ��塞�� �ணࠬ��� ��� �������� 䠩���.

"C:\Program" �� ����� ����७��� ��� ���譥�

Qt 6.7.3
```

**Comment by nlogozzo** [comment]
Interesting because it's seeing the locale as `ru-RU` but not translating the interface 🤔

I will look into it

**Comment by nlogozzo** [comment]
@besdar Can you try this build and see if it is fixed? https://github.com/NickvisionApps/Parabolic/actions/runs/11389664784/artifacts/2070233127

I wonder if it's an issue with gettext and Windows 10...others Windows users that use other locales have the UI correctly translated...

**Comment by besdar** [comment]
@nlogozzo it helped! Thank you! Now all UI is in Russian.

**Comment by nlogozzo** [comment]
> @nlogozzo it helped! Thank you! Now all UI is in Russian.

V2024.10.3 stable coming soon!

**Comment by nlogozzo** [comment]
The fix for this issue is now available in [V2024.10.3](https://github.com/NickvisionApps/Parabolic/releases/tag/2024.10.3) 🎉
